src/NBitcoin/Script.cs

```diff
@@ -691,7 +688,7 @@ public static uint256 SignatureHash(Script scriptCode, Transaction txTo, int nIn
             var scriptCopy = new Script(scriptCode._Script);
             scriptCopy.FindAndDelete(OpcodeType.OP_CODESEPARATOR);
 
-            var txCopy = new Transaction(txTo.ToBytes());
+            var txCopy = Transaction.Load(txTo.ToBytes(), network.Consensus.ConsensusFactory);
```

Comment:
Why is `.Load` is preferred over the normal constructor?

Reply:
Constructor is not used anymore, the factory will initialize the type.